oeps/architectural-decisions/oep-0026-arch-realtime-events.rst

```diff
@@ -28,7 +28,7 @@ OEP-26: Real-time Events
    :depth: 3
 
 Context
--------
+*******
```

Comment:
I think you mixed up `****` and `####` throughout this document (consistently, though: it seems to render fine :)